apis/core-api/src/api/middlewares/check-fan-club-existence/index.ts

```diff
@@ -0,0 +1,25 @@
+import { Middleware, Next as KoaNext } from 'koa';
```

Comment:
lets avoid this type of business layer separation,, keep the logic in controller or service please, in this way I wil suggest to write up to 100 middleware

Reply:
I need to throw exceptions in each service method is role does not exists?